com.unity.uiwidgets/Runtime/scheduler2/binding.cs

```diff
@@ -5,28 +5,29 @@
 using developer;
 using Unity.UIWidgets.async;
 using Unity.UIWidgets.async2;
+using Unity.UIWidgets.editor2;
```

Comment:
it is a mistake to put UIWidgetsPanelWrapper in the namespace Unity.UIWidgets.editor2. please do me a favor to change it to Unity.UIWidgets.engine2. Thanks!